kernel/api/src/main/java/org/sakaiproject/util/api/ValueEncryptionService.java

```diff
@@ -0,0 +1,21 @@
+package org.sakaiproject.util.api;
+
+public interface ValueEncryptionService {
```

Comment:
Ok, it's true we only want the Encryption Service inside our implementation. So, maybe it's a good idea to remove the service and left it only as a util.

We are thinking to rename it as CandidateDetailsProviderUtils and include the encrypt/decrypt methods and also all the static properties extracted from CandidateDetailProvider api https://github.com/ox-it/sakai/pull/67/files#diff-a06a3ab562dea9ac0026c28616a34faaR15 (as this util is going to be used only by our implementations). What do you think?